Dockerfile

```
@@ -0,0 +1,19 @@
+# elm-format doesn't work on node:lts-alpine or node:lts
+FROM node:lts-buster
+
+# Working directory as specified by exercism
+RUN mkdir /opt/representer
+RUN mkdir /opt/representer/bin
+WORKDIR /opt/representer
```

The following also works, as the `WORKDIR` command will automatically create the directory and the `COPY` command later on will create the `bin` directory.

```suggestion
WORKDIR /opt/representer
```